Generator/VisitsFake.php

```diff
@@ -143,7 +143,7 @@ public function generate($time = false, $idSite = 1, $limit = 1000)
                 $price =  $this->faker->randomNumber(2);
                 $quantity = $this->faker->numberBetween(1,4);
 
-                $tracker->addEcommerceItem($sku = $this->faker->randomNumber(4), $this->faker->productName, 'Music Category', $price, $quantity);
+                $tracker->addEcommerceItem($sku = $this->faker->randomNumber(1, 9999), $this->faker->productName, 'Music Category', $price, $quantity);
```

Comment:
`randomNumber(4)` creates a random number between `0` and `9999` (up to 4 digits), but as `0` is an invalid sku value, we need to ensure it's at least `1`